client/errs/errno.go

```diff
@@ -20,21 +20,20 @@ import (
 	"github.com/pingcap/errors"
 )
 
+// Note: keep the same as the ones defined on the server side to ensure the client can use them correctly.
 const (
+	// NoLeaderErr indicates there is no leader in the cluster currently.
+	NoLeaderErr = "no leader"
```

Comment:
When using this error?

Reply:
It can be returned here: https://github.com/tikv/pd/blob/199b01792159e5d8e83ef419a5053401e998bb0e/pkg/utils/apiutil/serverapi/middleware.go#L213